Octokit/Http/HttpClientAdapter.cs

```diff
@@ -45,7 +51,11 @@ public HttpClientAdapter(Func<HttpMessageHandler> getHandler)
                 // Make the request
                 var responseMessage = await _http.SendAsync(requestMessage, HttpCompletionOption.ResponseContentRead, cancellationTokenForRequest)
                                                 .ConfigureAwait(false);
-                return await BuildResponse(responseMessage).ConfigureAwait(false);
+                var response = await BuildResponse(responseMessage).ConfigureAwait(false);
+
+                LastApiInfo = response.ApiInfo;
```

Comment:
This does not have to be set here and IMO it's not a HTTP concern at all. All requests go trough [`Connection.RunRequest`](https://github.com/octokit/octokit.net/blob/a7bc09e76725cf053c4546e681def2c550796a64/Octokit/Http/Connection.cs#L522-L529) so I think it would be better to move this up a level.